src/Chapter2.hs

```diff
@@ -500,7 +506,10 @@ True
 >>> isThird42 [42, 42, 0, 42]
 False
 -}
-isThird42 = error "isThird42: Not implemented!"
+isThird42 :: (Eq a, Num a) => [a] -> Bool
+isThird42 []         = False
```

Comment:
This case is redundant, since it will be covered by the `_` pattern on the last line